### PR TITLE
fix: fix typos

### PR DESCRIPTION
**Pull request by rex4539**

### CODE REVIEW

**Comment by rex4539** [comment]
@cesarenaldi Could you take a look?

File: packages/api-bindings/src/lens/utils/open-actions.ts [hunk 1]
```diff
@@ -0,0 +1,209 @@
+/* eslint-disable no-case-declarations */
```
Comment (cesarenaldi) [inline packages/api-bindings/src/lens/utils/open-actions.ts:1]: I suspect this file was accidentally re-introduced while resolving a conflict.

**Review by cesarenaldi — CHANGES_REQUESTED** [review]
@rex4539 Thank you for taking the time.

In general all `packages/*-v1` are going to be deleted soon. No harm in your changes there, just letting you know.

Left a comment on a not expected file.

File: packages/api-bindings/src/lens/utils/open-actions.ts (hunk 1 above)
Comment (rex4539) [reply packages/api-bindings/src/lens/utils/open-actions.ts:1]: Removed.

**Comment by rex4539** [comment]
All should be good now.

**Review by cesarenaldi — APPROVED** [review]

**Pull request merged**